shopware/shopware/2017-06-22.yaml

```diff
@@ -0,0 +1,20 @@
+
+title:     Remote Code Execution Vulnerability
+link:      http://community.shopware.com/Security-Update-06-2017_detail_2014_482.html
```

Comment:
Link should be to english version: http://en.community.shopware.com/_detail_2015.html